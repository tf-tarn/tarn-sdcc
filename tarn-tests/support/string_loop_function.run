./sim/sim -v -q -s 5000 $program_file > $output
