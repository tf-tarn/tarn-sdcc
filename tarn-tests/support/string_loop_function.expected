Copy section of size 6 to 0x4000
Copy section of size 334 to 0x4006
Copy section of size 48 to 0x4154
Hello, world!
